packages/metadata/main/networks/list.cjs

```diff
@@ -3,6 +3,7 @@ module.exports =
 [
     {
         "name": "eth-goerli",
+        "isDeprecated": true,
```

Comment:
for actual deprecation, there needs a grace period. In absence of it, we either use a ticket to track, or we simply remove it here.

Reply:
Originally we said that eth-goerli will be removed by end of March, but that detail seems to have got lost.
fait accompli now, so will undo this.